From e3b78393d652a415a28ee0e3e684c6014219ac78 Mon Sep 17 00:00:00 2001
From: George Balatsouras <gbalats@gmail.com>
Date: Sun, 22 Jun 2014 13:51:02 +0300
Subject: [PATCH] Adding powerline stuff to tmux configuration.

---
 config/tmux.conf.bak | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/config/tmux.conf.bak b/config/tmux.conf.bak
index deab83d..7386042 100644
--- a/config/tmux.conf.bak
+++ b/config/tmux.conf.bak
@@ -28,3 +28,7 @@ bind-key \ split-window -h # split pane vertically
 
 # window names as titles (works well with tmuxinator)
 set-window-option -g allow-rename off
+
+# powerline setup
+source /usr/lib/python2.7/site-packages/powerline/bindings/tmux/powerline.conf
+set-option -g default-terminal "screen-256color"
-- 
1.9.3

